doc/languages-frameworks/rust.section.md

```diff
@@ -20,12 +20,16 @@ or use Mozilla's [Rust nightlies overlay](#using-the-rust-nightlies-overlay).
 Rust applications are packaged by using the `buildRustPackage` helper from `rustPlatform`:
 
 ```nix
-{ lib, rustPlatform }:
+{ lib, fetchFromGitHub, rustPlatform }:
```

Comment:
Oops, missed this one. Good catch! I think it's more common for the builder to go before fetchers, though.

Reply:
I'm usually sorting with {lib, stdenv, pkgs} in front and the sorted lexicographically. The real ripgrep expression also has this order.

Reply:
Apparently we actually have a rule for this!

> Arguments should be listed in the order they are used, with the exception of lib, which always goes first.

https://nixos.org/manual/nixpkgs/unstable/#sec-syntax

but since I already approved, I'm happy to merge as-is when OfBorg is happy.